### ISSUE TITLE
Dictionary Lookup not listing items  (  KR*GS but not KW*GS )

### ISSUE THREAD

**Issue by infodungeon** [openstenoproject/plover#917]
## Summary

Example;  "  shows KR*GS but not KW*GS even though both are listed in the dictionary

## Reproducing

Look up "  only one entry presented.
Not sure if this is a general or specific issue.

## Plover Version

Plover 4.0.0.dev7+10.gc4d2892 

## System

Window 10

**Comment by benoit-pierre** [comment]
Just to be clear:

* `"KR*GS": "{^\"}"` is shown
* but `"KR-GS": "{^~|\"}"` or `"KW-GS": "{~|\"^}"` are not

right?

**Comment by seebs** [comment]
Yeah, I see only KR*GS, and I know that KW-GS and KR-GS also exist, because those are the ones I actually use.

**Comment by zwagoth** [comment]
I am seeing this with entries like `really` which is in the dictionary as `R-L` which is getting assigned to `recall` which has `-RL`. I can see this in the 3.x branch of Plover as well, so it looks like a bigger issue with loading and normalizing strokes. Some entries are just outright deleted, such as `R-F` which maps to `rf` but it can't be found in the lookup tools by stroke or by translation.

I'll have a look and see if I can come up with anything more specific.
